otx/core/data/manager/dataset_manager.py

```diff
@@ -0,0 +1,142 @@
+
+    @staticmethod
+    def get_data_format(data_root: str) -> str:
+        """Find the format of dataset."""
+        data_root = os.path.abspath(data_root)
+
+        data_format: str = ""
+
+        # TODO #
+        # Currently, below `if/else` statements is mandatory
+        # because Datumaro can't detect the multi-cvat and mvtec.
+        # After, the upgrade of Datumaro, below codes will be changed.
+        if DatasetManager.is_cvat_format(data_root):
+            data_format = "multi-cvat"
+        elif DatasetManager.is_mvtec_format(data_root):
+            data_format = "mvtec"
+        else:
+            data_formats = datumaro.Environment().detect_dataset(data_root)
+            # TODO: how to avoid hard-coded part
+            data_format = data_formats[0] if "imagenet" not in data_formats else "imagenet"
+        return data_format
+
+    @staticmethod
+    def get_image_path(data_item: DatasetItem) -> str:
+        """Returns the path of image."""
+        return data_item.media.path
+
+    @staticmethod
+    def get_classification_label(data_item: DatasetItem) -> str:
+        """Returns the classfication label (multi-class)."""
+        assert len(data_item.annotations) == 1, "Not implemented for multi-label"
+        return data_item.annotations[0].label
+
+    @staticmethod
+    def export_dataset(dataset: Dataset, output_dir: str, data_format: str, save_media=True):
+        """Export the Datumaro Dataset."""
+        return Dataset.export(dataset, output_dir, data_format, save_media=save_media)
+
+    @staticmethod
+    def import_dataset(data_root: str, data_format: str) -> dict:
+        """Import dataset."""
+        return Dataset.import_from(data_root, format=data_format)
+
+    @staticmethod
+    def auto_split(task: str, dataset: Dataset, split_ratio: List[Tuple[str, float]]) -> dict:
+        """Automatically split the dataset: train --> train/val."""
+        splitter = Split(dataset, task.lower(), split_ratio)
+        return splitter.subsets()
+
+    @staticmethod
+    def is_cvat_format(path: str) -> bool:
+        """Detect whether data path is CVAT format or not.
+
+        Currently, we used multi-video CVAT format for Action tasks.
+
+        This function can detect the multi-video CVAT format.
+
+        Multi-video CVAT format
+        root
+        |--video_0
+            |--images
+                |--frame0001.png
+            |--annotations.xml
+        |--video_1
+        |--video_2
+
+        will be deprecated soon.
+        """
+
+        cvat_format = sorted(["images", "annotations.xml"])
+        for sub_folder in os.listdir(path):
+            # video_0, video_1, ...
+            sub_folder_path = os.path.join(path, sub_folder)
+            # files must be same with cvat_format
+            if os.path.isdir(sub_folder_path):
+                files = sorted(os.listdir(sub_folder_path))
+                if files != cvat_format:
+                    return False
+        return True
```

Comment:
`os.listdir` could show the proper error message in case of ex1, ex2.  So, I think we don't need to add exception handling.

Reply:
In my opinion, it's better that the function returns false at those cases. Do you think raising error is better?

Reply:
Yes, IMO, if the user gives an empty data directory or empty path, raising an error is better to analyze the root-cause because ex1, ex2 are kinds of abnormal cases